paper-drawer-panel.html

```diff
@@ -544,16 +574,18 @@
           this.selected = 'main';
         },
 
-        ready: function() {
-          // Avoid transition at the beginning e.g. page loads and enable
-          // transitions only after the element is rendered and ready.
-          this._transition = true;
-        },
-
-        _onMainTransitionEnd: function (e) {
-          if (e.currentTarget === this.$.main && (e.propertyName === 'left' || e.propertyName === 'right')) {
+        _onTransitionEnd: function (e) {
+          var target = Polymer.dom(e).localTarget;
+          if (target !== this) {
+            // ignore events coming from the light dom
+            return;
+          }
+          if (e.propertyName === 'left' || e.propertyName === 'right') {
             this.notifyResize();
           }
+          if (e.propertyName === 'transform' && this.selected === 'drawer' && this._focusedChild) {            
+            this._focusedChild.focus();
```

Comment:
Did you investigate if we still need to keep track of `this._focusedChild`, or can we just do `this._getAutoFocusedNode().focus()` here and below? (If so, might be worthwhile refactoring to `this._focusDrawerChild()`.)